server/src/server/session/TransactionOLTP.java

```diff
@@ -292,37 +279,29 @@ private static void mergeAttributeEdge(Vertex mergeTargetV, Vertex ent, GraphTra
         return propertiesAsObj.toArray();
     }
 
+    /**
+     * Creates a new Vertex in the graph and builds a VertexElement which wraps the newly created vertex
+     *
+     * @param baseType baseType of newly created Vertex
+     * @return VertexElement
+     */
     public VertexElement addVertexElement(Schema.BaseType baseType) {
-        return executeLockingMethod(() -> factory().addVertexElement(baseType));
+        Vertex vertex = janusTransaction.addVertex(baseType.name());
+        return factory().buildVertexElement(vertex);
     }
 
     /**
-     * This is only used when reifying a Relation
+     * This is only used when reifying a Relation, creates a new Vertex in the graph representing the reified relation.
+     * NB: this is only called when we reify an EdgeRelation - we want to preserve the ID property of the concept
      *
      * @param baseType  Concept BaseType which will become the VertexLabel
      * @param conceptId ConceptId to be set on the vertex
      * @return just created Vertex
      */
     public VertexElement addVertexElementWithEdgeIdProperty(Schema.BaseType baseType, ConceptId conceptId) {
-        return executeLockingMethod(() -> factory().addVertexElementWithEdgeIdProperty(baseType, conceptId));
-    }
-
-    /**
-     * Executes a method which has the potential to throw a TemporaryLockingException or a PermanentLockingException.
-     * If the exception is thrown it is wrapped in a GraknServerException so that the transaction can be retried.
-     *
-     * @param method The locking method to execute
-     */
-    private <X> X executeLockingMethod(Supplier<X> method) {
```

Comment:
removed this method as it adds complexity to code without any particular added value, first of all because there is no re-try policy anymore anywhere and the exception class is currently lost we reported to the client via gRPC.

The retry policy can be implemented by catching a BackendException from Janus where needed in the future.
And we should also figure out a way to report the correct exception type back to the client.